drracket-core-lib/drracket/private/language-object-contract.rkt

```diff
@@ -23,12 +23,12 @@
                                        (let loop ([s #'arg])
                                          (cond
                                            [(syntax? s) 
-                                            (let ([loc (vector (syntax-source s)
-                                                               (syntax-line s)
-                                                               (syntax-column s)
-                                                               (syntax-position s)
-                                                               (syntax-span s))])
-                                              (make-sloc (loop (syntax-e s)) loc))]
+                                            (vector (syntax-source s)
+                                                    (syntax-line s)
+                                                    (syntax-column s)
+                                                    (syntax-position s)
+                                                    (syntax-span s))
+                                            (make-sloc (loop (syntax-e s)) loc)]
```

Comment:
This doesn't look right.

Reply:
Ugh. Bitten by jackfirth/resyntax#449 again.